test/with_dummyserver/test_https.py

```diff
@@ -16,10 +17,13 @@
     requires_ssl_context_keyfile_password,
     resolvesLocalhostFQDN,
 )
+from test.conftest import ServerConfig
+from typing import List, Optional
 from unittest import mock
 
 import pytest
 import trustme
+from py._path.local import LocalPath
```

Comment:
`py._path` is private; the intended usage is `import py` and `path: py.path.local`.

BTW, some time in the future pytest is going to deprecate `py.path.local` fixtures (`tmpdir` and `tmpdir_factory`) in favor of pathlib fixtures (`tmp_path` and `tmp_path_factory`).

Reply:
Definitely, we only used py.path for Python 2 support.